tests/installation/user_settings.pm

```diff
@@ -27,12 +40,13 @@ sub run {
         return;
     }
 
-    send_key 'alt-f';    # Select full name text field
-    wait_screen_change { type_string $realname };
-    send_key 'tab';      # Select password field
-    send_key 'tab';
-    $self->type_password_and_verification;
-    assert_screen 'inst-userinfostyped';
+    $self->enter_userinfo();
+    if (!check_screen 'inst-userinfostyped') {
+        record_soft_failure 'bsc#1041747';
```

Comment:
sorry, I doubt this bug is a good choice. It's really powerPC-qemu specific. If there is no other bug yet then I suggest to create one.

@DimStar77 @nilxam @ggardet do you have a proper bug report, not a progress ticket about test issues, about the recent mistyping on openSUSE Tumbleweed?

Reply:
no we are not using bug detection needle, I'm going to delete those as we would need one needle for every case that can occur.